Package.swift

```diff
@@ -25,7 +28,13 @@ let package = Package(
         ),
         .testTarget(
             name: "PostHogTests",
-            dependencies: ["PostHog"],
+            dependencies: [
+                "PostHog",
+                "Quick",
+                "Nimble",
+                "OHHTTPStubs",
+                .product(name: "OHHTTPStubsSwift", package: "OHHTTPStubs"),
+            ],
```

Comment:
Here is what had to be done to make it work in CI, in combination with `swift test` instead of the xcode command for some reason.

Reply:
Xcode already defines those dependencies as SPM in its projects, but apparently, CI wasn't picking them up, maybe a Xcode version mismatch, an empty project worked just fine.